robottelo/ui/activationkey.py

```diff
@@ -59,6 +59,16 @@ def search_key_subscriptions(self, ak_name, subscription_name):
             (strategy, value % subscription_name))
         return element
 
+    def search_host(self, ak_name, host_name):
+        """Search for host activated by selected activation key."""
+        self.click(self.search(ak_name))
```

Comment:
We should probably improve our PR naming, as the PR with corresponding change was called "Added *** tests" so as i wasn't reviewer i never checked the code and had no idea new base helper was introduced.